src/Microsoft.SymbolManifestGenerator/SymbolManifestGenerator.cs

```diff
@@ -46,17 +46,22 @@ public static void GenerateManifest(ITracer tracer, DirectoryInfo dir, string ma
 
                     ManifestDataEntry manifestDataEntry = new()
                     {
-                        basedirRelativePath = basedirRelativePath,
-                        symbolKey = clrKey.Index,
-                        sha512 = fileHash,
-                        debugInformationLevel = Private
+                        BasedirRelativePath = basedirRelativePath,
+                        SymbolKey = clrKey.Index,
+                        Sha512 = fileHash,
+                        DebugInformationLevel = Private
                     };
 
-                    manifestData.entries.Add(manifestDataEntry);
+                    manifestData.Entries.Add(manifestDataEntry);
                 }
             }
 
-            string manifestDataContent = JsonSerializer.Serialize(manifestData);
+            JsonSerializerOptions serializeOptions = new()
+            {
+                PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
```

Comment:
Doesn't this break compatibility with the symbol.exe manifest?  Or does this option name the json properties as "entries", "version", "basedirRelativePath", etc.?

Reply:
The symbol.exe manifest structure is all camelCase, unlike what we have in this PR which was changed to all PascalCase.
So compatibility is ensured by using the CamelCase serializer options
example manifest that was generated
```json
{
  "entries": [
    {
      "basedirRelativePath": "runtimes\\win-arm64\\native\\mscordaccore.dll",
      "symbolKey": "<key>",
      "sha512": "<filehash",
      "debugInformationLevel": 340
    },
    {
      "basedirRelativePath": "runtimes\\win-arm64\\native\\mscordbi.dll",
      "symbolKey": "<key>",
      "sha512": "<filehash",
      "debugInformationLevel": 340
    },
    {
      "basedirRelativePath": "tools\\x64_arm64\\mscordaccore_amd64_arm64_9.0.24.28006.dll",
      "symbolKey": "<key>",
      "sha512": "<filehash",
      "debugInformationLevel": 340
    },
    {
      "basedirRelativePath": "runtimes\\win-arm64\\native\\mscordaccore_arm64_arm64_9.0.24.28006.dll",
      "symbolKey": "<key>",
      "sha512": "<filehash",
      "debugInformationLevel": 340
    }
  ],
  "version": "1"
}
```